src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java

```diff
@@ -77,6 +77,9 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
                 .then(literal("stop")
                     .requires(source -> source.hasPermissionLevel(2))
                     .executes(GameCommand::stopGame)
+                        .then(literal("confirm")
+                        .requires(source -> source.hasPermissionLevel(2))
```

Comment:
I believe the parenthesis is wrong here- the `executes(stopGameConfirmed)` will be attached to the root stop node

Reply:
It works in game and you can reformat it to 
```java
then(
    literal("stop")
        .requires(source -> source.hasPermissionLevel(2))
        .executes(GameCommand::stopGame)
                .then(
                    literal("confirm")
                    .requires(source -> source.hasPermissionLevel(2))
                        .executes(GameCommand::stopGameConfirmed)
                )
)
```

Reply:
Oops, ignore me